app/components/Modals/VotingModal/VotingModal.jsx

```diff
@@ -0,0 +1,246 @@
+// @flow
+import React, { Component } from 'react'
+
+import BaseModal from '../BaseModal'
+import Button from '../../Button'
+import styles from './styles.scss'
+import classNames from 'classnames'
+
+type Props = {
+  hideModal: Function,
+  showSuccessNotification: Function
+}
+
+type State = {
+  infoShowing: boolean,
+  confirmationShowing: boolean,
+  votes: Object
+}
+
+const textStrings = {
+  TITLE: 'Vote for consensus node',
+  TOP_10_CANDIDATES: 'Top 10 candidates',
+  CANDIDATES: 'Candidates',
+  NUMBER_OF_VOTES: 'Number of votes',
+  CAST_VOTE: 'Cast vote',
+  EXPLANATION_TITLE: 'How does voting work?',
+  VOTING_EXPLANATION: 'Each NEO node can vote for the candidtates. The number of NEON in the current voting account will be automatically calculated as the number of the candidate\'s votes. When voting for multiple candidates, each candidate gets the votes equal to the NEO number of the current voting account. For example, if there are 100 NEO in the current account and the three candidates are voted for from this account, each candidate receives 100 votes. If NEO in the account is spent after the vote, the candidates\' votes will simultaneously be decreased to the current NEO balance',
+  PUBLIC_ADDRESS: 'Public address of candidate',
+  CONFIRMATION_TITLE: 'Confirm votes',
+  CONFIRMATION_MESSAGE: 'You will be casting <number_of_votes> votes, for each of the following candidates:',
+  SUBMIT: 'Submit',
+  VOTE_SUCCESS: 'Vote successfully submitted!'
+}
+
+const placehoderData = [
+  {
+    address: 'AXmzKD3dvj7dPUQKkBeNZmRDYF6AhrwrtQ',
+    votes: 111111111
+  },
+  {
+    address: 'AXmzKD3dvj7dPUQKkBeNZmRDYF6AhrwrtQ',
+    votes: 222222222
+  },
+  {
+    address: 'AXmzKD3dvj7dPUQKkBeNZmRDYF6AhrwrtQ',
+    votes: 333333333
+  },
+  {
+    address: 'AXmzKD3dvj7dPUQKkBeNZmRDYF6AhrwrtQ',
+    votes: 444444444
+  },
+  {
+    address: 'AXmzKD3dvj7dPUQKkBeNZmRDYF6AhrwrtQ',
+    votes: 555555555
+  },
+  {
+    address: 'AXmzKD3dvj7dPUQKkBeNZmRDYF6AhrwrtQ',
+    votes: 666666666
+  },
+  {
+    address: 'AXmzKD3dvj7dPUQKkBeNZmRDYF6AhrwrtQ',
+    votes: 777777777
+  },
+  {
+    address: 'AXmzKD3dvj7dPUQKkBeNZmRDYF6AhrwrtQ',
+    votes: 888888888
+  },
+  {
+    address: 'AXmzKD3dvj7dPUQKkBeNZmRDYF6AhrwrtQ',
+    votes: 101010101
+  }
+]
+
+const votesAvailable = 123
+
+export default class VotingModal extends Component<Props, State> {
+  state = {
+    infoShowing: false,
+    confirmationShowing: false,
+    votes: {}
+  }
+
+  showInfoModal: Function
+  hideInfoModal: Function
+  handleNodeSelected: Function
+  handleCastVote: Function
+  showConfirmationModal: Function
+  hideConfirmationModal: Function
+  handleVoteSubmit: Function
+  hasVotes: Function
+
+  componentWillMount () {
+    this.showInfoModal = this.showInfoModal.bind(this)
```

Comment:
You wrote these so you could better test it? Because you could just use es6 arrow functions, and you won't need to bind.

Also, if you do happen to want to use this, then place it in the constructor